Makefile

```diff
@@ -1,5 +1,11 @@
+GIT_SHA := $(shell git rev-parse --short HEAD 2>/dev/null)
+GIT_TAG := $(shell git describe --abbrev=0 HEAD 2>/dev/null)
+LD_FLAGS := '-s -w \
+	-X main.versionTag=$(GIT_TAG)-$(GIT_SHA) \
+	-X main.bashCompletion=$(shell base64 -w0 bash-completion)'
+
 build:
-	go build -o wfv ./cmd/...	
+	go build -ldflags=$(LD_FLAGS) -trimpath ./cmd/...
```

Comment:
unfortunately this is not working on my machine as intended

Reply:
```
which-fix-version git:(shell-magic-completion-retore) make build
base64: invalid option -- w
Usage:	base64 [-hvDd] [-b num] [-i in_file] [-o out_file]
  -h, --help     display this message
  -Dd, --decode   decodes input
  -b, --break    break encoded string into num character lines
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
go build -ldflags='-s -w -X main.versionTag=-27b9af3 -X main.bashCompletion=' -trimpath ./cmd/...
```

Reply:
ye, this is gnu version of base64 wich has -wN to say how long a line may be. If you dont have gnu base64, then you can/must replace newlines with empty string in the output, as this is the requirement to bake in the file in such a way.

Alternatively the file can be baked into the binary with this approach:

```
import _  embed

//go:embed version.txt
var s string
```

where version.txt is a local file in the src structure. This approach requires recent golang.